### PR TITLE
Fixes Black Mesa security guard preview text

### PR DESCRIPTION
**Pull request by FelixBlet**
## About The Pull Request

so, there be a problem in description on mesa, guards have some text of scientists and i fix it
its pretty small update

(screenshot of this problem: [boop!](https://media.discordapp.net/attachments/574255116371230720/1022797450609692692/unknown.png))

## How This Contributes To The Skyrat Roleplay Experience

now, when u wanna play on guards in first time (or not in first time)? u dont see strange message like: u are scientist and u need survive in dark scary place!!!!

## Changelog

edit some black mesa code
nothing more

:cl:
fix: fixed a description of guards on Black Meas
/:cl:

### CODE REVIEW

File: modular_skyrat/modules/black_mesa/code/ghost_spawners.dm [hunk 1]
```diff
@@ -37,6 +37,10 @@
 	prompt_name = "a research facility guard"
 	outfit = /datum/outfit/security_guard
 	you_are_text = "You are a security guard in a top secret government facility. You blacked out. Now, you have woken up to the horrors that lay within. DO NOT TRY TO EXPLORE THE LEVEL. STAY AROUND YOUR AREA."
+	flavour_text = "You are a security guard in a top secret government facility. You blacked out. Now, you have woken up to the horrors that lay within."
+
+
+
 
 /obj/effect/mob_spawn/ghost_role/human/black_mesa/guard/special(mob/living/carbon/human/spawned_human)
```
Comment (ghost) [inline modular_skyrat/modules/black_mesa/code/ghost_spawners.dm:45]:
```suggestion
	flavour_text = "You are a security guard in a top secret government facility. You blacked out. Now, you have woken up to the horrors that lay within."

/obj/effect/mob_spawn/ghost_role/human/black_mesa/guard/special(mob/living/carbon/human/spawned_human)
```
Only need 1 line of whitespace

**Comment by ghost** [comment]
Title should usually be more descriptive (such as `Fixes Black Mesa security guard preview text`) so reviewers know what to expect when they open a PR

Also you did a typo in the CL, Mesa not Meas :b

File: modular_skyrat/modules/black_mesa/code/ghost_spawners.dm (hunk 1 above)
Comment (FelixBlet) [reply modular_skyrat/modules/black_mesa/code/ghost_spawners.dm:45]: i love big whitespace, sorry

**Comment by Lukaster42** [comment]
#16396
also fixes this problem.

**Comment by Stalkeros2** [comment]
Yeah we've already fixed that.

File: modular_skyrat/modules/black_mesa/code/ghost_spawners.dm (hunk 1 above)
Comment (ghost) [reply modular_skyrat/modules/black_mesa/code/ghost_spawners.dm:45]: (So do I, its why I commented it - better to break the habit before it becomes an issue lmao)

**Comment by FelixBlet** [comment]
oh, 'k

**Pull request closed without merging**